en/services/high_latency.md

```diff
@@ -1,27 +1,29 @@
 # High Latency Protocol
 
-High latency links (e.g. Iridium Satellite links) provide global connectivity, albeit with significant message latency and high cost-per-message.
-Generally the cost and latency means that high-latency links are only used when there is no lower-latency alternative, and when active the links should only send essential information.
+High latency links (e.g. Iridium Satellite links) provide global connectivity, albeit with significant message latency (> 1 second) and high cost-per-message.
+Generally the cost and latency means that high-latency links are only used when there is no lower-latency alternative, and when active the links should only send essential information or commands.
 
-The protocol provides a heartbeat-like message ([HIGH_LATENCY2](#HIGH_LATENCY2)) for transmitting just the most important telemetry at low rate, and a command ([MAV_CMD_CONTROL_HIGH_LATENCY](#MAV_CMD_CONTROL_HIGH_LATENCY)) for enabling/disabling the high latency link when needed (i.e. when no lower-latency link is available).
+The protocol provides a heartbeat-like message ([HIGH_LATENCY2](#HIGH_LATENCY2)) for transmitting just the most important telemetry at low rate, and a command ([MAV_CMD_CONTROL_HIGH_LATENCY](#MAV_CMD_CONTROL_HIGH_LATENCY)) for enabling/disabling the high latency link when needed (i.e. when no lower-latency link is available). Alternatively, the flight controller can be set to enable the high latency link by default (i.e. all links are active throughout the flight for redundancy purposes).
 
 ## Heartbeat/Routing
 
 High latency links are expensive.
 In order to reduce traffic to the bare minimum, some of the fundamental assumptions of MAVLink are explicitly broken:
-- [HEARTBEAT](../messages/common.md#HEARTBEAT) messages are not emitted on the channel (either by the autopilot or GCS).
+- [HEARTBEAT](../messages/common.md#HEARTBEAT) messages are not emitted on the channel by the autopilot. The GCS will continue to send the heatbeat to the autopilot though.
 
-  > **Note** The heartbeat is used to build MAVLInk routing tables between channels.
+  > **Note** The heartbeat is used to build MAVLink routing tables between channels.
     Commands addressed specifically to the high latency component may not be routed from another channel (i.e. you can connect to the component from a GCS directly, but not via a MAVLink router).
-- Broadcast messages from the network should not be automatically sent over the high latency channel.
+- Only the [COMMAND_LONG](../messages/common.md#COMMAND_LONG) and [MAV_CMD_ACK](../messages/common.md#MAV_CMD_ACK) messages should be sent over the high latency channel.
+- The GCS should send the [HEARTBEAT](../messages/common.md#HEARTBEAT) message, in order to allow for loss-of-datalink failsafes to work.
+- Ground stations should not request full sets of waypoints or parameters in order to prevent congestion.
```

Comment:
As well as this, we should capture the normal use intent as per https://github.com/mavlink/mavlink-devguide/pull/368#issuecomment-867070835

So set up connection when in LL, switch to HL and off you go.

Reply:
Heartbeats are never sent GCS->Vehicle on a high latency link. The reason being high latency links tend to be expensive things like sat links. Way too expensive dollar-wise to send heartbeats in that direction.

Reply:
Added normal user intent in the "Sequences" section